<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1403</width>
    <height>1044</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" rowspan="2">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>2</horstretch>
        <verstretch>2</verstretch>
       </sizepolicy>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
      <property name="dragMode">
       <enum>QGraphicsView::NoDrag</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <layout class="QVBoxLayout" name="verticalLayout_5"/>
    </item>
    <item row="0" column="1" rowspan="2">
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Heroes:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="lbl_hero_count">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Monsters:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="lbl_monster_count">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Villagers:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="lbl_villager_count">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Scheusals:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="lbl_scheusal_count">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>dead:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="lbl_deadcount">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="2">
         <widget class="QPushButton" name="pushButton_11">
          <property name="text">
           <string>Stop</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="pushButton_5">
          <property name="text">
           <string>Up</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>Left</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="pushButton_4">
          <property name="text">
           <string>Right</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>Down</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton_18">
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="8" column="0" colspan="2">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="pushButton_16">
          <property name="text">
           <string>Music On</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_17">
          <property name="text">
           <string>Music Off</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="9" column="0" colspan="2">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Zoom</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="scaleSlider">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>500</number>
          </property>
          <property name="value">
           <number>250</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="6" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButton_14">
          <property name="text">
           <string>Spawn All</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_15">
          <property name="text">
           <string>Clear Field</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_12">
          <property name="text">
           <string>Start All</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_13">
          <property name="text">
           <string>Stop All</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QFormLayout" name="formLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Timer:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="lbl_timer_interval">
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_10">
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_9">
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1403</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionThis_is_a_test">
   <property name="text">
    <string>This is a test</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_16</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>musicOn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1324</x>
     <y>823</y>
    </hint>
    <hint type="destinationlabel">
     <x>1315</x>
     <y>747</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_17</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>musicOff()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1300</x>
     <y>868</y>
    </hint>
    <hint type="destinationlabel">
     <x>1304</x>
     <y>907</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>changeVolume(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1350</x>
     <y>917</y>
    </hint>
    <hint type="destinationlabel">
     <x>1366</x>
     <y>957</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_14</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>spawnAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1326</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>1122</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>scaleSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>scaleScene(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>28</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>27</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>spriteDown()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1300</x>
     <y>605</y>
    </hint>
    <hint type="destinationlabel">
     <x>702</x>
     <y>993</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_15</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clearActors()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1310</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>1063</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>spriteUp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1300</x>
     <y>505</y>
    </hint>
    <hint type="destinationlabel">
     <x>847</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_12</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>startScene()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1270</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>1023</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>spriteLeft()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1260</x>
     <y>555</y>
    </hint>
    <hint type="destinationlabel">
     <x>638</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_10</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>spriteSpeedSlow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1280</x>
     <y>385</y>
    </hint>
    <hint type="destinationlabel">
     <x>940</x>
     <y>983</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_13</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>stopScene()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1280</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>1111</x>
     <y>397</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_18</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>spriteStart()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1290</x>
     <y>465</y>
    </hint>
    <hint type="destinationlabel">
     <x>1087</x>
     <y>987</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>spriteRight()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1340</x>
     <y>555</y>
    </hint>
    <hint type="destinationlabel">
     <x>739</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_9</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>spriteSpeedFast()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1260</x>
     <y>385</y>
    </hint>
    <hint type="destinationlabel">
     <x>802</x>
     <y>997</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_11</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>spriteStop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1290</x>
     <y>425</y>
    </hint>
    <hint type="destinationlabel">
     <x>952</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setLineEdit()</slot>
  <slot>drawScene()</slot>
  <slot>spriteDown()</slot>
  <slot>spriteLeft()</slot>
  <slot>spriteRight()</slot>
  <slot>spriteUp()</slot>
  <slot>addHero()</slot>
  <slot>addMonster()</slot>
  <slot>addVillager()</slot>
  <slot>addScheusal()</slot>
  <slot>spriteSpeedFast()</slot>
  <slot>spriteSpeedSlow()</slot>
  <slot>spriteStop()</slot>
  <slot>stopScene()</slot>
  <slot>startScene()</slot>
  <slot>spawnAll()</slot>
  <slot>clearActors()</slot>
  <slot>musicOn()</slot>
  <slot>musicOff()</slot>
  <slot>spriteStart()</slot>
  <slot>addMountain()</slot>
  <slot>addStream()</slot>
  <slot>changeVolume(int)</slot>
  <slot>slot2()</slot>
  <slot>scaleScene(int)</slot>
 </slots>
</ui>
